File: LeetCode/Easy/Remove Vowels From String/Go/remove_vowels_from_string.go
package main

import "fmt"

func remove_vowels_from_string(s string) string {
	str := []byte{}
	for i := 0; i < len(s); i++ {
		if s[i] == 'a' || s[i] == 'e' || s[i] == 'i' || s[i] == 'o' || s[i] == 'u' {
			continue
		}
		str = append(str, s[i])
	}
	return string(str)
}

func main() {
	fmt.Println(remove_vowels_from_string("leetcodeisacommunityforcoders"))
}

File: LeetCode/Easy/Remove Vowels From String/Cpp/remove_vowels_from_string.cc
#include <iostream>

std::string remove_vowels_from_string(std::string s) {
    std::string str = "";
    for (int i=0; i<s.size(); i++) 
        if (!(s[i] == 'a' || s[i] == 'e' || s[i] == 'i' || s[i] == 'o' || s[i] == 'u')) str += s[i];
    return str;
}

int main() {
    std::cout<<remove_vowels_from_string("leetcodeisacommunityforcoders")<<"\n";
}
